stdlib/public/core/Character.swift

```diff
@@ -353,6 +353,14 @@ public struct Character :
   internal var _representation: Representation
 }
 
+extension Character : CustomStringConvertible {
```

Comment:
Oh, that's because `LosslessStringConvertible` refines `CustomStringConvertible`.